supplementary_materials/qwen2/convert.py

```diff
@@ -0,0 +1,74 @@
+from transformers import AutoTokenizer, AutoConfig
+from optimum.intel import OVWeightQuantizationConfig
+from optimum.intel.openvino import OVModelForCausalLM
+
+import os
+from pathlib import Path
+import argparse
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument('-h',
+                        '--help',
+                        action='help',
+                        help='Show this help message and exit.')
+    parser.add_argument('-m',
+                        '--model_id',
+                        default='Qwen/Qwen1.5-0.5B-Chat',
+                        required=False,
+                        type=str,
+                        help='orignal model path')
+    parser.add_argument('-p',
+                        '--precision',
+                        required=False,
+                        default="int4",
+                        type=str,
+                        choices=["fp16", "int8", "int4"],
+                        help='fp16, int8 or int4')
+    parser.add_argument('-o',
+                        '--output',
+                        default='./Qwen1.5-0.5B-ov',
```

Comment:
can we do not hardcode output dir and change it base on model_id if not provided? (it may lead to some confusion if they are not aligned)